File: help/pt_pt/exercicios_da_ficha_9/Ficha09_008/Ficha09_008.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Soma da sequência de fibonacci\n " ;
    $f0 = 1 ;
    $f1 = 1 ;
    echo $f0  .  " " . $f1  ;
    $soma = 2 ;
    while($f0 + $f1 < 100 ){
        $fn = $f0 + $f1 ;
        $soma = $soma + $fn ;
        echo  " " . $fn  ;
        $f1 = $f0 ;
        $f0 = $fn ;
    }
    echo  "\nSoma = " . $soma  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Thu, 2022 Sep 29 12:25

?>

File: help/pt_pt/exercicios_da_ficha_9/Ficha09_008/Ficha09_008.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

public class Ficha09_008 {

    //Main Function
    public static void main( String[] args) {
        System.out.print("Soma da sequência de fibonacci\n " );
        int f0 = 1 ;
        int f1 = 1 ;
        System.out.print(f0 + " " + f1 );
        int soma = 2 ;
        while(f0 + f1 < 100 ){
            int fn = f0 + f1 ;
            soma +=fn;
            System.out.print(" " + fn );
            f1 = f0 ;
            f0 = fn ;
        }
        System.out.print("\nSoma = " + soma );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Thu, 2022 Sep 29 12:25
